packages/kit/src/runtime/server/constants.js

```diff
@@ -1 +1,5 @@
+import { DEV } from 'esm-env';
+import process from 'node:process';
+
 export const NULL_BODY_STATUS = [101, 103, 204, 205, 304];
+export const IN_WEBCONTAINER = DEV && !!process.versions.webcontainer;
```

Comment:
Sometimes I need to reproduce things in `vite preview` too. Would removing the `DEV` check so that it works in StackBlitz make sense in this case?
```suggestion
export const IN_WEBCONTAINER = !!process.versions.webcontainer;
```

Reply:
It should be fine, but I'm not sure.